assets/js/blocks/product-categories/block.js

```diff
@@ -2,73 +2,138 @@
+		this.renderList = this.renderList.bind( this );
+		this.renderOptions = this.renderOptions.bind( this );
+	}
 
-	const renderList = ( items ) => (
-		<ul key={ parentKey }>
-			{ items.map( ( cat ) => {
-				const count = hasCount ? <span>({ cat.count })</span> : null;
-				return [
-					<li key={ cat.term_id }>
-						<a href={ isPreview ? null : cat.link }>{ cat.name }</a> { count } { /* eslint-disable-line */ }
-					</li>,
-					!! cat.children && !! cat.children.length && renderList( cat.children ),
-				];
-			} ) }
-		</ul>
-	);
+	onNavigate() {
+		const { isPreview = false } = this.props;
+		const url = this.select.current.value;
+		if ( 'false' === url ) {
+			return;
+		}
+		const home = wc_product_block_data.homeUrl;
 
-	return (
-		<div className="wc-block-product-categories">
-			{ isDropdown ? (
-				<SelectControl
-					label={ __( 'Select a category', 'woo-gutenberg-products-block' ) }
-					options={ categories.map( ( cat ) => ( {
-						label: hasCount ? `${ cat.name } (${ cat.count })` : cat.name,
-						value: cat.term_id,
-					} ) ) }
-					onChange={ noop }
-				/>
-			) : (
-				renderList( categories )
-			) }
-		</div>
-	);
-};
+		if ( ! isPreview && 0 === url.indexOf( home ) ) {
+			document.location.href = url;
+		}
+	}
+
+	renderList( items, depth = 0 ) {
+		const { isPreview = false } = this.props;
+		const { hasCount } = this.props.attributes;
+		const parentKey = 'parent-' + items[ 0 ].term_id;
+
+		return (
+			<ul key={ parentKey }>
+				{ items.map( ( cat ) => {
+					const count = hasCount ? <span>({ cat.count })</span> : null;
+					return [
+						<li key={ cat.term_id }>
+							<a href={ isPreview ? null : cat.link }>{ cat.name }</a> { count } { /* eslint-disable-line */ }
+						</li>,
+						!! cat.children && !! cat.children.length && this.renderList( cat.children, depth + 1 ),
+					];
+				} ) }
+			</ul>
+		);
+	}
+
+	renderOptions( items, depth = 0 ) {
+		const { hasCount } = this.props.attributes;
+
+		return items.map( ( cat ) => {
+			const count = hasCount ? `(${ cat.count })` : null;
+			return [
+				<option key={ cat.term_id } value={ cat.link }>
+					{ repeat( '–', depth ) } { cat.name } { count }
+				</option>,
+				!! cat.children && !! cat.children.length && this.renderOptions( cat.children, depth + 1 ),
+			];
+		} );
+	}
+
+	render() {
+		const { attributes, instanceId } = this.props;
+		const { isDropdown } = attributes;
+		const categories = getCategories( attributes );
+		const classes = classnames( {
+			'wc-block-product-categories': true,
+			'is-dropdown': isDropdown,
+			'is-list': ! isDropdown,
+		} );
+
+		const selectId = `prod-categories-${ instanceId }`;
+
+		return (
+			<div className={ classes }>
+				{ isDropdown ? (
+					<Fragment>
+						<label className="screen-reader-text" htmlFor={ selectId }>
+							{ __( 'Select a category', 'woo-gutenberg-products-block' ) }
+						</label>
+						<select id={ selectId } ref={ this.select }>
+							<option value="false">
```

Comment:
What do you think about making it hidden so it doesn't appear as an option in the dropdown?
```suggestion
							<option value="false" hidden>
```

Reply:
Sure – looks like this only works in Chrome & Firefox, but it doesn't have any negative effect in the other browsers 👍